### PR TITLE
Feartures/help message

### PR DESCRIPTION
**Pull request by Mikanebu**
* added help response for command `\ourbot help` pointing to the `README.md` usage section
* added `resolver` repo into list of repos

### CODE REVIEW

File: extras/closeMilestones.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ let repos = [
   {owner: 'datahq', repo: 'assembler'},
   {owner: 'datahq', repo: 'pm'},
   {owner: 'datahq', repo: 'specstore'},
+  {owner: 'datahq', repo: 'resolver'},
```
Comment (rufuspollock) [inline extras/closeMilestones.js:17]: This commit would be better as a simple push outside of this pull request (it is not part of the changes described in this PR).

But not a biggie!

File: README.md [hunk 2]
```diff
@@ -9,7 +9,12 @@ Our Hubot-based bot to automate lots of stuff including:
 
 # How to use it
 
-In your chat channel tag at start, middle or end of a message:
+To get `help` command, run the following code:
+`ourbot help` - to get general help message information
+
+`/ourbot help` - to see available functionalities of `ourbot`. 
+
+In your chat channel tag at start, middle or end of a message: `+todo`, `+standup`, `+examples`
```
Comment (rufuspollock) [inline README.md]: This should have a heading e.g. `## TODOs` so that it can be specifically linked to from the help

File: scripts/main.js [hunk 3]
```diff
@@ -35,6 +35,9 @@ module.exports = (robot) => {
     robot.hear(/badger/i, (res) => {
         res.send("Badgers? BADGERS? WE DON'T NEED NO STINKIN BADGERS")
     })
+    robot.hear(/ourbot help/i, (res) => {
+        res.send("Hi, I'm your helpful chatops bot! Please, see README for the usage https://github.com/datopian/ourbot#how-to-use-it")
```
Comment (rufuspollock) [inline scripts/main.js]: This should list the main commands available and link them to the README as we discussed when planning this work.

**Review by rufuspollock — CHANGES_REQUESTED** [review]

File: scripts/main.js (hunk 3 above)
Comment (Mikanebu) [reply scripts/main.js]: @rufuspollock Yes, it was created yesterday, i modified as you suggested. Trying to write unittest. Btw, you can check now writing `bot` or `/bot` to get a response

File: Procfile [hunk 4]
```diff
@@ -1 +1 @@
-web: HUBOT_GITTER2_TOKEN=$HUBOT_GITTER2_TOKEN bin/hubot -a gitter2 --name ourbot
```
Comment (rufuspollock) [inline Procfile]: This should be part of a separate commit or change - it is not part of this PR 😄

File: test/parsing-test.js [hunk 5]
```diff
@@ -30,7 +30,22 @@ describe('Messages parsing', function () {
         sendGst.restore()
         getRoom.restore()
     })
-
+    it('starting with bot should reply help message', function () {
```
Comment (rufuspollock) [inline test/parsing-test.js]: No, this is not correct. I would suggest it only replies with help message if the message starts with bot and only contains bot or you have bot help e.g.

```
bot
bot help
/bot
/bot help
```

Anything else should not generate a help message.

Comment (rufuspollock) [reply test/parsing-test.js]: And great we have these tests so we can see this!

File: test/parsing-test.js [hunk 6]
```diff
@@ -81,35 +96,35 @@ describe('Messages parsing', function () {
...
 
     it('Without any tag', function () {
-        room.user.say('weirdguy', "do this one").then(function () {
+        return room.user.say('weirdguy', "do this one").then(function () {
             assert.equal(sendMsg.callCount, 0)
             assert.equal(sendGst.callCount, 0)
         })
     })
-
+
```
Comment (rufuspollock) [inline test/parsing-test.js]: Why do we have whitespace here ...

File: test/parsing-test.js [hunk 7]
```diff
@@ -81,35 +96,35 @@ describe('Messages parsing', function () {
...
+        return room.user.say('weirdguy', "do this one").then(function () {
             assert.equal(sendMsg.callCount, 0)
             assert.equal(sendGst.callCount, 0)
         })
     })
-
+    
     it('Tag in middle', function () {
-        room.user.say('weirdguy', "do +todo this one").then(function () {
+        return room.user.say('weirdguy', "do +todo this one").then(function () {
```
Comment (rufuspollock) [inline test/parsing-test.js]: Why are we returning this now vs before?

File: test/parsing-test.js (hunk 5 above)
Comment (Mikanebu) [reply test/parsing-test.js]: @rufuspollock will be fixed

**Pull request merged**